lib/assertions/matches-selector.ts

```diff
@@ -0,0 +1,15 @@
+export default function matchesSelector(
+  targetSelector: string | Element,
+  compareSelector: string
+): Array<number> {
+  let targetElements: Array<Element> =
+    typeof targetSelector === 'string'
+      ? Array.from(document.querySelectorAll(targetSelector))
+      : [targetSelector];
```

Comment:
this code should probably reuse the `findElements()` method of the `Assertions` class. I also think we should just pass an `Element[]` array to this function, and let the surrounding code worry about dealing with the target selectors.

Reply:
I applied the suggested changes.